### PR TITLE
Initial Azure agent check-in

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

File: azure/exports/service.template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=WALA Agent
+Wants=network-online.target sshd.service sshd-keygen.service
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/python  -u /usr/sbin/waagent -daemon
+Restart=always
+RestartSec=5
+WorkingDirectory=${DESTDIR/rootfs
```
Comment (cgwalters) [inline azure/exports/service.template]: `}` ?

File: azure/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM centos:latest
+RUN yum -y install WALinuxAgent && yum -y clean all
+COPY exports /
```
Comment (cgwalters) [inline azure/Dockerfile]: Missing?

Comment (baude) [inline azure/Dockerfile]: I dont understand your comment.  Can you elaborate ?

Comment (yuqi-zhang) [inline azure/Dockerfile]: The current syntax is copying a file called exports into the container's root, which you don't have included. I think for the time being what you want is `COPY service.template /exports/`

File: azure/exports/service.template (hunk 1 above)
Comment (baude) [inline azure/exports/service.template]: Fixed

File: azure/Dockerfile (hunk 2 above)
Comment (baude) [inline azure/Dockerfile]: @cgwalters ok now ?

**Comment by giuseppe** [comment]
@yuqi-zhang, @cgwalters does the new version look fine now?

**Comment by yuqi-zhang** [comment]
Files seem fine to me, but just the Dockerfile and the service file is not enough to get it working, so I'm not sure if we should just merge as is as a "check-in" or leave it open until its ready?

**Comment by giuseppe** [comment]
@yuqi-zhang it doesn't use runc.  It spawns a python program so the config.json.template file is not necessary

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 3d9116d

**Comment by yuqi-zhang** [comment]
@giuseppe I see, just that I was testing on centosAH and it error'ed with  `/usr/bin/python: can't open file '/usr/sbin/waagent': [Errno 2] No such file or directory`, so I was wondering if it was missing some bindings or tmpfiles.template

**Comment by giuseppe** [comment]
@baude, should it be '-u usr/sbin/waagent'? /usr is still the one from the host. Alternatively it can be $DESTDIR/rootfs/usr/sbin/waagent'

**Comment by giuseppe** [comment]
@baude are you still on it or would you prefer if I open a PR for the issue up here?

**Pull request closed without merging**